src/containers/observation/page/index.js

```diff
@@ -102,25 +105,74 @@ export default class ObservationPage extends React.Component {
     })
   }
 
-  getLabel(key) {
-    return this.props.translate(`musit.observation.page.${key}`)
-  }
-
   typeDefinitions = {
     '': { label: 'typeSelect.labelText' },
-    gas: { label: 'gas.labelText', render: this.renderDoubleTextArea },
-    lightConditions: { label: 'lux.labelText', render: this.renderDoubleTextArea },
-    cleaning: { label: 'cleaning.labelText', render: this.renderDoubleTextArea },
-    pest: { label: 'pest.labelText', render: this.renderPest, data: ObservationPage.createDefaultPestData() },
-    mold: { label: 'mold.labelText', render: this.renderDoubleTextArea },
-    skallsikring: { label: 'skallsikring.labelText', render: this.renderDoubleTextArea },
-    tyverisikring: { label: 'tyverisikring.labelText', render: this.renderDoubleTextArea },
-    brannsikring: { label: 'brannsikring.labelText', render: this.renderDoubleTextArea },
-    vannskaderisiko: { label: 'vannskaderisiko.labelText', render: this.renderDoubleTextArea },
-    relativeHumidity: { label: 'rh.labelText', render: this.renderFromToNumberComment },
-    hypoxicAir: { label: 'hypoxicAir.labelText', render: this.renderFromToNumberComment },
-    temperature: { label: 'temperature.labelText', render: this.renderFromToNumberComment },
-    alcohol: { label: 'alcohol.labelText', render: this.renderAlcohol }
+    gas: {
+      label: 'gas.labelText',
+      render: this.renderDoubleTextArea,
+      validate: validation.validateDoubleTextArea
+    },
+    lightConditions: {
+      label: 'lightConditions.labelText',
+      render: this.renderDoubleTextArea,
+      validate: validation.validateDoubleTextArea
+    },
+    cleaning: {
+      label: 'cleaning.labelText',
+      render: this.renderDoubleTextArea,
+      validate: validation.validateDoubleTextArea
+    },
+    pest: {
+      label: 'pest.labelText',
+      render: this.renderPest,
+      data: ObservationPage.createDefaultPestData(),
+      validate: validation.validatePest
+    },
+    mold: {
+      label: 'mold.labelText',
+      render: this.renderDoubleTextArea,
+      validate: validation.validateDoubleTextArea
+    },
+    skallsikring: {
```

Comment:
Change the language.  Please find below the list of backend keys(you can remove the Observation keyword from key)  
ObservationAlcohol
ObservationCleaning
ObservationFireProtection
ObservationGas
ObservationHypoxicAir
ObservationLightingCondition
ObservationMold
ObservationPerimeterSecurity
ObservationRelativeHumidity
ObservationPest
ObservationTemperature
ObservationTheftProtection
ObservationWaterDamageAssessment